testsuite/Gemfile.lock

```diff
@@ -0,0 +1,170 @@
+GEM
```

Comment:
We shouldn't commit this file.

Reply:
Agreed. I have removed it and instead fix the dependencies in the Gemfile.